pyproject.toml

```diff
@@ -65,6 +65,7 @@ editable-backend = "editables"
 [tool.pdm.dev-dependencies]
 duty = ["duty>=0.7"]
 docs = [
+    "markdown-callouts>=0.2.0",
```

Comment:
```suggestion
    "markdown-callouts>=0.2.0",
    "mkdocs>=1.3",
```

Reply:
It wouldn't hurt, but MkDocs is already a prod-dependency so is this really useful? I personally never install the docs group without the default one.

Reply:
Ah, yeah, makes sense. Thanks.